storybookjs/storybook
# HTML: Add CSF types

## What I did

This PR does the following:

1. Add 6.0 types to @storybook/html
2. Add HTML typescript CLI templates
3. A kitchen sink project in the examples folder for HTML typescript

## How to test

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? Added!
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
